src/types/UiaTextRangeBase.cpp

```diff
@@ -317,7 +317,7 @@ try
     const auto sensitivity = ignoreCase ? Search::Sensitivity::CaseInsensitive : Search::Sensitivity::CaseSensitive;
 
     auto searchDirection = Search::Direction::Forward;
-    auto searchAnchor = _start;
+    COORD searchAnchor = _start;

```

Comment:
these seem like superfluous changes. `auto` makes it _even easier_ to become til::point later.